amazon_transcribe/utils.py

```diff
@@ -38,3 +43,19 @@ def ensure_boolean(val):
         return val
     else:
         return val.lower() == "true"
+
+async def apply_realtime_delay(
+    stream: AudioStream,
+    reader: aiofile.Reader,
```

Comment:
I missed this in the last review. We don't currently have a dependency on aiofile. Let's change this to an Iterable<bytes> which is more correct with what we're expecting here. It just happens that the Reader implements this behavior.